embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/EmbraceImpl.kt

```diff
@@ -304,14 +306,16 @@ internal class EmbraceImpl @JvmOverloads constructor(
      * Shuts down the Embrace SDK.
      */
     fun stop() {
-        if (sdkCallChecker.started.compareAndSet(true, false)) {
-            logger.logInfo("Shutting down Embrace SDK")
-            runCatching {
-                application?.let {
-                    unregisterComposeActivityListener(it)
+        synchronized(sdkCallChecker) {
```

Comment:
This should not happen unless the SDK is shutting down, so locking isn't that big of a deal